packages/terra-date-time-picker/src/terra-dev-site/doc/guides/DateTimePickerEvents.jsx

```diff
@@ -65,7 +65,7 @@ class DateTimePickerEvents extends React.Component {
           <span id="onBlur-count">{this.state.blurTriggerCount}</span>
           <br />
           <br />
-          onClickOutside Trigger Count:
+          onDismiss Trigger Count:
           {' '}
           <span id="onClickOutside-count">{this.state.clickOutsideTriggerCount}</span>
```

Comment:
Rename `clickOutsideTriggerCount`, `clickOutsideCount`, `onClickOutside-count`, etc. This file can be cleaned up a bit by using the `useState` hook.

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1224/commits/0d52d24371525054cdc6c2c7a5749c68b4b43901